src/Equations/acoustic/Model/Datastructures.h

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file
+ * This file is part of SeisSol.
+ *
+ * @author Carsten Uphoff (c.uphoff AT tum.de,
+ *http://www5.in.tum.de/wiki/index.php/Carsten_Uphoff,_M.Sc.)
+ * @author Sebastian Wolf (wolf.sebastian AT in.tum.de,
+ *https://www5.in.tum.de/wiki/index.php/Sebastian_Wolf,_M.Sc.)
+ *
+ * @section LICENSE
+ * Copyright (c) 2015 - 2020, SeisSol Group
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ *    contributors may be used to endorse or promote products derived from this
+ *    software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * @section DESCRIPTION
+ **/
+
+#ifndef MODEL_ACOUSTIC_DATASTRUCTURES_H_
+#define MODEL_ACOUSTIC_DATASTRUCTURES_H_
+
+#include "Model/CommonDatastructures.h"
+#include "generated_code/init.h"
+#include "generated_code/kernel.h"
+#include <array>
+#include <cmath>
+#include <cstddef>
+#include <string>
+
+namespace seissol::model {
+struct AcousticMaterial : Material {
+  static constexpr std::size_t NumQuantities = 4;
+  static constexpr std::size_t NumberPerMechanism = 0;
+  static constexpr std::size_t Mechanisms = 0;
+  static constexpr MaterialType Type = MaterialType::Acoustic;
+  static constexpr LocalSolver Solver = LocalSolver::CauchyKovalevski;
+  static inline const std::string Text = "acoustic";
+  static inline const std::array<std::string, NumQuantities> Quantities = {
```

Comment:
If we are sticking to "p" here, are you using the original pressure or negative of pressure in line with the stress-velocity formulation? This needs to be clearly mentioned as otherwise it would be really confusing when we later couple the elastic-acoustic formulation.